google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -960,6 +960,34 @@ def schema
           s
         end
 
+        ##
+        # Creates a new encryption configuration instance.
+        #
+        # @param [String] kms_key Name of the Cloud KMS encryption key that
+        #   will be used to protect destination BigQuery table. The BigQuery
```

Comment:
"protect destination" -> "protect the destination"

Reply:
Done in https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2016/commits/ac778399167536e4e610f2d899f0d1c7f2b4415a